### ISSUE TITLE
Add ggpairs wrapper

### ISSUE THREAD

**Issue by mark-andrews** [mark-andrews/psyntur#11]
Make wrapper for `ggpairs` to make a scatterplot matrix with histograms or density plots on diagonal.

**Comment by jensroes** [comment]
Check the pairs plot in this app: https://jens-roeser.shinyapps.io/multilangapp/
I think adding density plots isn't too difficult. It was tricky to get the axes text on the right places.

**Comment by mark-andrews** [comment]
I merged in your #21 into the development branch, but there is a problem with the `columns` call in `ggpairs`. It is probably easy to fix, but the preferable thing would be use nse for the variables, e.g. 

```r
pairs_plot(variables = c(sex_dimorph, attractive), data = faithfulfaces)
```

But I find nse is bloody difficult and so I couldn't quickly figure immediately so I didn't do it immediately. But I think I know now. The nice thing is that it would also allow us to do eg. 

```r
pairs_plot(variables = starts_with('foo'), data = faithfulfaces)
```

and other tidy selection methods :)

**Comment by mark-andrews** [comment]
This was dealt with some time ago, but I did not close the issue and I forget which commit dealt with it too....sorry
